docs/src/prototypes/CopyToClipboard/NotificationProvider.tsx

```diff
@@ -28,9 +35,21 @@ export const NotificationProvider: React.FC = props => {
 
   return (
     <>
-      <Portal open={!!notification}>
-        <Notification>{notification}</Notification>
-      </Portal>
+      {!!targetRef ? (
+        !!notification && (
+          <Tooltip
+            content={notification}
+            pointing={true}
+            open={!!notification}
+            target={targetRef.current}
```

Comment:
This is dangerous, let's use `target`:

```suggestion
            target={targetRef}
```

Reply:
wouldn't that be inconsistent with Popup component where we have
```
target?: HTMLElement
```